examples/documentation_examples/examples.go

```diff
@@ -1971,7 +1971,10 @@ func WithTransactionExample(ctx context.Context) error {
 	defer func() { _ = client.Disconnect(ctx) }()
 
 	// Prereq: Create collections.
-	wcMajority := writeconcern.New(writeconcern.WMajority(), writeconcern.WTimeout(1*time.Second))
+	wcMajority := &writeconcern.WriteConcern{
```

Comment:
Can we use the convenience function in the example, instead of using the "majority" magic string? 

```go
wcMajority := writeconcern.Majority()
wcMajority.WTimeout = 1 * time.Second
```